Create docathon page for docathon 2021 (#786)
mattermost/mattermost-handbook: operations/research-and-development/product/technical-writing/community-documentation-process.md

# Community Documentation

> In the past, community-managed documentation lived in the product documentation. In some cases, this created some confusion as although the content was in our product documentation it wasn't written and tested by Mattermost and thus not officially supported.

Reviewer: ```suggestion
In the past, community-managed documentation lived in the product documentation. This created some confusion because, although the content was in our product documentation, it wasn't written and tested by Mattermost and thus not officially supported.
```

> Additionally, although the content may still need testing and we encourage feedback, having it hosted in a static site didn't lend itself to engagement and interaction.

Reviewer: ```suggestion
Additionally, having the content hosted on a static site didn't lend itself to engagement and interaction.
```

> Going forward, our existing community-written documentation will be posted in the dedicated forum category.

Reviewer: ```suggestion
Going forward, our existing community-written documentation will be available on the [Mattermost Forum](https://forum.mattermost.org/) within a dedicated forum category called **Unofficial Product Documentation**.
```

## Process for new community-written content

> If you've written, or would like to write, new content for Mattermost please add it as a new thread in the [Unofficial Product Documentation](https://forum.mattermost.org/c/docs/37) category.

Reviewer: ```suggestion
If you've written, or want to write, new content for Mattermost, please add it as a new thread in the [Unofficial Product Documentation](https://forum.mattermost.org/c/docs/37) category.
```
